src/main/java/com/jfrog/ide/common/configuration/JfrogCliDriver.java

```diff
@@ -146,6 +147,30 @@ public void downloadCliFromReleases(Version cliVersion, String destinationFolder
         }
     }
 
+    public void addCliServerConfig(String xrayUrl, String artifactoryUrl, String cliServerId, String user, String password, String accessToken, File workingDirectory) throws IOException {
+        List<String> args = new ArrayList<>();
+        args.add("config");
+        args.add("add");
+        args.add(cliServerId);
+        args.add("--xray-url=" + xrayUrl);
+        args.add("--artifactory-url=" + artifactoryUrl);
+        args.add("--interactive=false");
+        args.add("--overwrite");
+
+        if (accessToken != null && !accessToken.isEmpty()) {
+            args.add("--access-token=" + accessToken);
+        } else {
+            args.add("--user=" + user);
+            args.add("--password=" + password);
+            args.add("--enc-password=false");
+        }
+
+        try {
+            runCommand(workingDirectory, args.toArray(new String[0]), Collections.emptyList(), new NullLog());
```

Comment:
```suggestion
            runCommand(workingDirectory, args.toArray(new String[0]), Collections.emptyList(), log);
```
please pass the JFrogCliDriver log, so we can get logs of the executed command